app/components/post_draft/post_draft.js

```diff
@@ -373,16 +471,42 @@ export default class PostDraft extends PureComponent {
         }
     };
 
+    mapGroupMentions = (groupMentions) => {
+        const {channelMemberCountsByGroup} = this.props;
+        let memberNotifyCount = 0;
+        let channelTimezoneCount = 0;
+        const groupMentionsSet = new Set();
+        groupMentions.
+            forEach((group) => {
+                const mappedValue = channelMemberCountsByGroup[group.id];
+                if (mappedValue?.channel_member_count > NOTIFY_ALL_MEMBERS && mappedValue?.channel_member_count > memberNotifyCount) {
+                    memberNotifyCount = mappedValue.channel_member_count;
+                    channelTimezoneCount = mappedValue.channel_member_timezones_count;
+                }
+                groupMentionsSet.add(`@${group.name}`);
+            });
+        return {groupMentionsSet, memberNotifyCount, channelTimezoneCount};
+    }
+
     sendMessage = () => {
         const value = this.input.current?.getValue() || '';
-        const {enableConfirmNotificationsToChannel, membersCount, useChannelMentions} = this.props;
+        const {enableConfirmNotificationsToChannel, membersCount, useGroupMentions, useChannelMentions} = this.props;
         const notificationsToChannel = enableConfirmNotificationsToChannel && useChannelMentions;
+        const notificationsToGroups = enableConfirmNotificationsToChannel && useGroupMentions;
         const toAllOrChannel = this.textContainsAtAllAtChannel(value);
+        const groupMentions = (!toAllOrChannel && notificationsToGroups) ? this.groupsMentionedInText(value) : [];
 
         if (value.indexOf('/') === 0) {
             this.sendCommand(value);
         } else if (notificationsToChannel && membersCount > NOTIFY_ALL_MEMBERS && toAllOrChannel) {
             this.showSendToAllOrChannelAlert(membersCount);
+        } else if (notificationsToGroups && !toAllOrChannel && groupMentions.length > 0) {
```

Comment:
`groupMentions` is already  checking for `!toAllOrChannel && notificationsToGroups` so here you  can change it to
```suggestion
        } else if (groupMentions.length > 0) {
```